frontend/services/popupNotifications.service.js

```diff
@@ -0,0 +1,74 @@
+(function () {
+  "use strict";
+
+  angular
+    .module("base")
+    .service("popupNotifications", popupNotifications);
+
+  popupNotifications.$inject = ["Alertify"];
+
+  function popupNotifications(Alertify) {
+    var vm = this;
```

Comment:
in services it is better to use another alias, for example service. VM isn't correct, because we don't have access to view